### DISCUSSION TITLE
is auto-combine a possibility?

### DISCUSSION THREAD

**Discussion by tily-lass** [DFHack/dfhack#3029] [Ideas]
the "combine all" command is amazing, and a life saver... is there a way to "autocombine all", maybe once every other day or something?

**Comment by adiamaq** [comment]
How does autocombine works and what does it do? o.o

**Reply by tily-lass** [reply]
use the "combine all" command - it combines stacks of drinks, meat, plants, meals, etc. so instead of having, say, 7 barrels with stacks of 1-2 biscuits, you will have one with, say, 20

**Comment by myk002** [comment]
Yes, it is easy to set up running a command on a schedule, relative to game calendar time. This is essentially what all the items on the Maintenance tab in `gui/control-panel` are doing. `combine` could be added to that list.
